migrations/sqls/20250630125251-returns-id-up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE returns ADD COLUMN id UUID DEFAULT gen_random_uuid();
+
+UPDATE returns SET id = gen_random_uuid();
+
+ALTER TABLE returns ALTER COLUMN id SET NOT NULL;
```

Comment:
Any migration we add will have a comment above the migration to explain why we're making the change.

See, the previous 5 migrations for an example.

I'd suggest copying your PR description is a good enough for this instance.

On the migration, same comment about applies.

```suggestion
ALTER TABLE "returns"."returns" ADD COLUMN id UUID DEFAULT gen_random_uuid();

UPDATE "returns"."returns" SET id = gen_random_uuid();

ALTER TABLE "returns"."returns" ALTER COLUMN id SET NOT NULL;
```